bts_phot/calibrate_fps.py

```diff
@@ -386,19 +397,28 @@ def get_baseline(fps_file, window="10D",
                         chi = 1/(len(mask[0])-1)*sum_diff_sq
                         mean = np.mean(non_det)
                         median = np.median(non_det)
+                        
+                        medians = np.empty(1000)
+                        for i in range(1000): 
+                            medians[i] = np.median(np.random.choice(non_det, 
+                                         size = len(non_det), replace = True))
+                        median_unc = np.diff(np.percentile(medians, (16,84)))[0]/2
+                        
                         low_limit = np.percentile(non_det, 10)
                         high_limit = np.percentile(non_det, 90)
                         trim = np.where((non_det > low_limit) &
                                         (non_det < high_limit)
                                         )
                         trim_mean = np.mean(non_det[trim])
                         if len(non_det[trim]) == 0:
-                            print(f'{ztf_name} {ufid} pre empty')
+                            print(f'{ztf_name} {ufid} post empty')
```

Comment:
Are you sure this is post? we set the value of pre right after this

Reply:
you are right, I forgot to change it after copying & pasting